helm/ffc-pay-enrichment/values.yaml

```diff
@@ -19,17 +19,12 @@ containerSecret:
   type: Opaque
 
 deployment:
```

Comment:
Can see why these are not required via here: https://github.com/DEFRA/ffc-helm-library/blob/master/ffc-helm-library/templates/_deployment.yaml